R/wordlist.R

```diff
@@ -15,12 +15,12 @@
 #' @family spelling
 #' @export
 #' @param confirm show changes and ask confirmation before adding new words to the list
+#' @param pkg path to package (or bookdown folder)
 #' @inheritParams spell_check_package
 update_wordlist <- function(pkg = ".", vignettes = TRUE, confirm = TRUE){
-  pkg <- as_package(pkg)
-  wordfile <- get_wordfile(pkg$path)
-  old_words <- sort(get_wordlist(pkg$path))
-  new_words <- sort(spell_check_package(pkg$path, vignettes = vignettes, use_wordlist = FALSE)$word)
+  wordfile <- get_wordfile(pkg)
+  old_words <- sort(get_wordlist(pkg))
+  new_words <- sort(spell_check_package(pkg, vignettes = vignettes, use_wordlist = FALSE)$word)
```

Comment:
how could one make this bookdown-compatible too? another update_wordlist() function?